external/fv3fit/fv3fit/train_microphysics.py

```diff
@@ -204,6 +208,93 @@ def __post_init__(self) -> None:
             self.cache = False
 
 
+def get_model_output_sensitivities(
+    model: tf.keras.Model, sample: Mapping[str, tf.Tensor],
```

Comment:
this function should work for any tensorflow code...so best to pose broadly.
```suggestion
    model: Callable[[Mapping[str, tf.Tensor]], Mapping[str, tf.Tensor]], sample: Mapping[str, tf.Tensor],
```